app/helpers/application_helper.rb

```diff
@@ -8,7 +8,7 @@ def abbr_tag(i18n_key, options = {})
   end
 
   def page_title
-    return "#{@page_title} — #{t('site.name')}" if @page_title
+    return "#{ActionView::Base.full_sanitizer.sanitize(@page_title)} — #{t('site.name')}" if @page_title
```

Comment:
This will scrub any HTML from `@page_title` (since the `<title>` element expects plain text). So now we can do things like:

```erb
<%- @page_title = "#{abbr_tag('dod')} Civilian Employees" -%>
```

…which will render appropriately in an `<h1>` on the page and in the navigation breadcrumbs. It'll also appear as the plain text "DOD Civilian Employees" in the `<title>` element.